config/config.json

```diff
@@ -39,6 +39,11 @@
       "basePath": "/mas-local-fm/smart_events",
       "proxyTarget": "http://0.0.0.0:9006",
       "fallbackBasePath": "/beta/apps/se-ui-build"
+    },
+    "sas_ui": {
+      "basePath": "/mas-local-fm/sas_ui",
+      "proxyTarget": "http://0.0.0.0:9000",
+      "fallbackBasePath": "/beta/apps/rhosak-control-plane-ui-build"
```

Comment:
This will have to change based on how the SA build process will be configured. Just a reminder.

Reply:
I'll keep that in mind